istio-operator/istio-operator.spec

```diff
@@ -64,16 +64,10 @@ mkdir -p OPERATOR/src/github.com/maistra/istio-operator
 tar zxf %{SOURCE0} -C OPERATOR/src/github.com/maistra/istio-operator --strip=1
 
 %build
-if [[ "%{community_build}"  == "true" ]]; then
-  export BUILD_TYPE="maistra"
-else
-  export BUILD_TYPE="servicemesh"
-fi
-
 cd OPERATOR
 export GOPATH=$(pwd):%{gopath}
 pushd src/github.com/maistra/istio-operator/
-VERSION=%{version}-%{release} GITREVISION=%{git_shortcommit} GITSTATUS=Clean GITTAG=%{version} make compile collect-resources
+COMMUNITY=%{community_build} GO111MODULE=on VERSION=%{version}-%{release} GITREVISION=%{git_shortcommit} GITSTATUS=Clean GITTAG=%{version} make compile collect-resources collect-olm-manifests
```

Comment:
`collect-resources` task should include (require) `collect-olm-manifests`.

Reply:
I thought about that, but `collect-resources` seems to refer to the resources directory which this is not part of so I kept them separate.